applications/MACI-CDao.md

```diff
@@ -0,0 +1,268 @@
+
+Sam, MBA, Senior Project Manager, Data Analyst, with over 20 years of cross disciplinary experience Project experience, CDao initiator.
+
+### Team Code Repos
+
+- [ SuperFairGovernance ](https://github.com/SuperFairGovernance) 
+
+## Development Status :  
+
+  The application is in response to the [anti-collusion infrastructure RFP](https://grants.web3.foundation/docs/RFPs/anti-collusion_infrastructure). 
+
+  Development status will be found over at [super-fair-chain](https://github.com/iunknow588/super-fair-chain). The anti-collusion infrastructure  will be implemented by several pallets components of the super fair chain.
+
+## Development Roadmap :  
+
+### Overview
+
+- **Total Estimated Duration:**    3 months
+- **Full-Time Equivalent (FTE):**  1.5 FTE
+- **Total Costs:**                 28,000 USD
+- **DOT %:**                       50%
+
+### Milestone 1 :  Pallet logic — Basic functionality
+
+- **Estimated duration:** 1 month
+- **FTE:**                1
+- **Costs:**              8,000 USD 
+ 
+- Description: Pallet written in Rust for Substrate 2.0 based blockchains. The Pallet will use decentralized decision algorithms to implement the Minimum Anti-Collusion Infrastructure (MACI) mechanism. 
+
+| Number | Deliverable    | Specification | 
+| ------ | -----------    | ------------- |
+| 0a.    | License        | Apache 2.0    |  
+| 0b.    | Documentation  | Code documentation, API reference |  
+| 0c.    | Tests          | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide, we will describe how to run these tests. |  
+| 1.     | Usage example  |  Example of how to use the pallet inside code  |    
+
+
+### Milestone 2  Pallet logic — Auxiliary functions features
+
+- **Estimated Duration:** 1 month
+- **FTE:**                1.5
+- **Costs:**              10,000 USD 
+- Description: Pallet written in Rust for Substrate 2.0 based blockchains. The Pallet will implement  member management and bit commitment voting process,together with decentralized decision pallets, it forms the Minimum anti-collusion Infrastructure (MACI).   
+
+| Number | Deliverable    | Specification | 
+| ------ | -----------    | ------------- |
+| 0a.    | License        | Apache 2.0    |  
+| 0b.    | Documentation  | Code documentation, API reference |  
+| 0c.    | test           | The code will have unit-test coverage (min.70%) to ensure functionality and robustness. In the guide, we will describe how to run these tests. |  
+| 1.     |  Usage example | Example of how to use the pallet inside code.|   
+| 2.     | anti-collusion | Implement a mechanism to prevent bribery and collusion, leveraging encrypting votes potentially via Minimum anti-collusion Infrastructure (MACI) | 
```

Comment:
This sounds very vague. It would be nice if you could commit harder in the specification of the deliverables to what you want to implement.

Reply:
Thank you very much for pointing out this issue. I will revise it later